app/pb_kits/playbook/pb_background/docs/_background_default.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= pb_rails("background", props: { background_color: "bg_gradient" }) do %>
+  Background Kit Gradient Color
+<% end %>
+
+<%= pb_rails("background", props: { background_color: "bg_dark" }) do %>
+  <%= pb_rails("flex", props: { horizontal: "center"}) do %>
+    <%= pb_rails("flex/flex_item") do %>
+      <%= pb_rails("title", props: { dark: true, size: 2, text: "Background Kit Dark" })%>
+    <% end %>
+  <% end %>
+<% end %>
+
+<%= pb_rails("background", props: { background_color: "bg_light", padding: "xl" }) do %>
+  <%= pb_rails("flex", props: { horizontal: "right"}) do %>
+    <%= pb_rails("flex/flex_item") do %>
+      <%= pb_rails("body", props: { text: "Background Kit Light" })%>
+    <% end %>
+  <% end %>
+<% end %>
+
+<%= pb_rails("background", props: { background_color: "white", padding: "lg" }) do %>
+  <%= pb_rails("flex") do %>
+    <%= pb_rails("flex/flex_item") do %>
+      <%= pb_rails("title", props: { size: 3, text: "Background Kit White" })%>
+    <% end %>
+  <% end %>
+<% end %>
+
+<%= pb_rails("background", props: { image_url: "https://www.proflowers.com/blog/wp-content/uploads/2016/04/hero-blue-flowers.jpg", padding: "lg" }) do %>
+  <%= pb_rails("flex", props: { horizontal: "center", orientation: "column" }) do %>
+    <%= pb_rails("flex/flex_item") do %>
+      <%= pb_rails("title", props: { dark: true, size: 1, text: "Background Kit Image" })%>
+    <% end %>
+    <%= pb_rails("flex/flex_item", props: { padding: "lg" }) do %>
+      <%= pb_rails("card") do %>
```

Comment:
Can we use a card with no border + shadow (it will look better I think)

`border_none: true, shadow: "deepest"`

Reply:
we left out bored none because that is the default (: